server.js

```diff
@@ -149,15 +150,26 @@ LimitdServer.prototype.start = function (done) {
   return this;
 };
 
-LimitdServer.prototype.stop = function () {
+LimitdServer.prototype.stop = function (callback) {
   var self = this;
   var log = self._logger;
   var address = self._server.address();
+  callback = cb(callback || _.noop).timeout(5000).once();
+  log.debug(address, 'closing server');
 
-  this._server.destroy(function() {
+  this._server.destroy(() => {
```

Comment:
Seems that there are not many instances when this could emit an error. But still I think we should handle it when node is updated or so, this could end up emiting more errors that we will swallow (it calls server.close + destroys the connections behind scenes)